src/current/_includes/v25.2/known-limitations/physical-cluster-replication.md

```diff
@@ -1,3 +1,6 @@
-- Physical cluster replication is supported in CockroachDB {{ site.data.products.core }} clusters on v23.2 or later. The primary cluster can be a [new]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#step-1-create-the-primary-cluster) or [existing]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#set-up-pcr-from-an-existing-cluster) cluster. The standby cluster must be a [new cluster started with the `--virtualized-empty` flag]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#step-2-create-the-standby-cluster).
+- Physical cluster replication is supported in:
```

Comment:
suggest code formatting for the `--virtualized-empty` flag unless maybe it broke something?